### PR TITLE
Feature/refactor eventsbutton

### PR DESCRIPTION
**Pull request by mdvanes**
Changed the delimiters
Refactored to Atoms
Added Storybook stories

### CODE REVIEW

File: src/components/Atoms/EventsButton/EventsButton.stories.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```
Comment (mts) [inline src/components/Atoms/EventsButton/EventsButton.stories.js]: React is warning about a unique key prop for this story.

File: src/components/Atoms/EventsButton/EventsButton.stories.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import { storiesOf } from '@storybook/react';
```
Comment (mts) [inline src/components/Atoms/EventsButton/EventsButton.stories.js]: Maybe this is more like a molecule now that it's wrapping Tooltip and CustomButton.

File: src/components/Molecules/NavBar/NavBar.stories.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```
Comment (mts) [inline src/components/Molecules/NavBar/NavBar.stories.js]: NavBar and also TeamCarousel look nice on Storybook.

File: src/components/Atoms/EventsButton/EventsButton.stories.js (hunk 2 above)
Comment (mdvanes) [reply src/components/Atoms/EventsButton/EventsButton.stories.js]: Agreed, this is still difficult to guess correctly for me at the moment.

File: src/components/Atoms/EventsButton/EventsButton.stories.js (hunk 1 above)
Comment (mdvanes) [reply src/components/Atoms/EventsButton/EventsButton.stories.js]: I have seen this, but I have trouble reproducing it. It only seems to do it in Storybook and only sometimes (refreshing the browser). Do you have a suggestion of what would need a key?

**Pull request merged**